com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumeAsset.cs

```diff
@@ -50,6 +75,86 @@ public string GetSerializedFullPath()
+
+            static int AlignUp16(int count)
+            {
+                var alignment = 16;
+                var remainder = count % alignment;
+                return count + (remainder == 0 ? 0 : alignment - remainder);
+            }
+
+            LoadBinaryData();
+
+            var cellData = assetToBytes[cellDataAsset];
+            var bricksByteCount = totalCellCounts.bricksCount * UnsafeUtility.SizeOf<ProbeBrickIndex.Brick>();
+            var shDataByteStart = AlignUp16(bricksByteCount);
+            var shDataByteCount = totalCellCounts.probesCount * UnsafeUtility.SizeOf<Vector4>() * CoefficientVectorsCount;
+            var bricksData = cellData.GetSubArray(0, bricksByteCount).Reinterpret<ProbeBrickIndex.Brick>(1);
+            var shData = cellData.GetSubArray( shDataByteStart, shDataByteCount).Reinterpret<Vector4>(1);
+
+            var cellSupportData = cellSupportDataAsset ? assetToBytes[cellSupportDataAsset] : default;
+            var hasSupportData = cellSupportData.IsCreated;
+            var positionsByteCount = totalCellCounts.probesCount * UnsafeUtility.SizeOf<Vector3>();
+            var validityByteStart = AlignUp16(positionsByteCount);
+            var validityByteCount = totalCellCounts.probesCount * UnsafeUtility.SizeOf<float>();
+            var offsetByteStart = AlignUp16(positionsByteCount) + AlignUp16(validityByteCount);
+            var offsetByteCount = totalCellCounts.offsetsCount * UnsafeUtility.SizeOf<Vector3>();
+            var positionsData = hasSupportData ? cellSupportData.GetSubArray(0, positionsByteCount).Reinterpret<Vector3>(1) : default;
+            var validityData = hasSupportData ? cellSupportData.GetSubArray(validityByteStart, validityByteCount).Reinterpret<float>(1) : default;
+            var offsetsData = hasSupportData ? cellSupportData.GetSubArray(offsetByteStart, offsetByteCount).Reinterpret<Vector3>(1) : default;
+
+            var startCounts = new CellCounts();
+            for (var i = 0; i < cells.Length; ++i)
+            {
+                var cell = cells[i];
+                var counts = cellCounts[i];
+
+                cell.bricks = bricksData.GetSubArray(startCounts.bricksCount, counts.bricksCount);
+                cell.shData = shData.GetSubArray(startCounts.probesCount * CoefficientVectorsCount, counts.probesCount * CoefficientVectorsCount);
+
+                if (hasSupportData)
+                {
+                    cell.probePositions = positionsData.GetSubArray(startCounts.probesCount, counts.probesCount);
+                    cell.validity = validityData.GetSubArray(startCounts.probesCount, counts.probesCount);
+                    cell.offsetVectors = offsetsData.GetSubArray(startCounts.offsetsCount, counts.offsetsCount);
+                }
+
+                startCounts.Add(counts);
+            }
+        }
+
+        // TEMP SECTION: Manually extract a NA from bytes until we have access to: https://github.cds.internal.unity3d.com/unity/unity/pull/6458
```

Comment:
Reminder for me/you to remove when we have that PR landed.

Reply:
Done.